src/KOKKOS/fix_enforce2d_kokkos.h

```diff
@@ -56,7 +56,8 @@ class FixEnforce2DKokkos : public FixEnforce2D {
 
 
 template <class DeviceType, int omega_flag, int angmom_flag, int torque_flag>
-struct FixEnforce2DKokkosPostForceFunctor {
```

Comment:
I thought this code is the exactly same--i.e. a struct defaults to public members, while a class defaults to private. What was the warning here?

Reply:
The warning was
```
warning: 'FixNVEKokkosInitialIntegrateFunctor' defined as a struct template here but previously declared as a class template; this is valid, but may result in linker errors under the Microsoft C++ ABI [-Wmismatched-tags]
```
Even though a struct in C++ **behaves** like a class with all public methods, I would consider it more consistent to not assume they are identical at the ABI level. I give you that Windows is always a bit "special", but - like with getting rid of the warnings in general - I think it is preferable to be a bit more defensive and explicit.

Reply:
I see--the declaration in the header was "class" but the implementation in the .cpp was "struct" so they didn't match. My preference would be to switch everything to "struct" rather than "class", I can do that.

Reply:
Certainly; for as long as it is consistent...

Reply:
There is a bunch more than the ones in this PR. I recall having made similar changes before.